helpers/handlebarsHelpers.js

```diff
@@ -13,10 +13,10 @@ module.exports = {
     return new Handlebars.SafeString(gaCode);
   },
   addOpenGraph (title, mimeType, showUrl, source, description, thumbnail) {
-    if (title === null || title.trim() === '') {
+    if (!title || title.trim() === '') {
```

Comment:
Same with this - let's sanitize data sooner. Ideally, the data could be sanitized in both cases _before_ it even makes it into the DB. Otherwise, you should clean it as close as you can to where it is retrieved. It'll make your code easier to write and will prevent introduction of bugs caused by forgetting to do a trim check.